client/dashboard/domains/domain-connection-setup/components/switch-setup.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import { DomainConnectionSetupMode } from '@automattic/api-core';
+import { Meta, StoryObj } from '@storybook/react';
+import { StepType, StepName } from '../types';
+import SwitchSetup from './switch-setup';
+
+const meta: Meta< typeof SwitchSetup > = {
+	title: 'client/dashboard/domains/domain-connection-setup/SwitchSetup',
+	component: SwitchSetup,
+	tags: [ 'autodocs' ],
+	parameters: {
+		actions: { argTypesRegex: '^on.*' },
+	},
+	argTypes: {
+		setPage: { action: 'setPage' },
+	},
+};
+
+export default meta;
+
+type Story = StoryObj< typeof SwitchSetup >;
+
+export const SuggestedModeWithDomainConnect: Story = {
```

Comment:
Same. I'm not sure what's the benefits of these stories. Instead, it seems better to write the unit test to verify the logics.